bindings/storage-catches.i

```diff
@@ -311,6 +311,7 @@
 %catches(storage::Exception) storage::Partitionable::get_default_partition_table_type() const;
 %catches(storage::WrongNumberOfChildren, storage::DeviceHasWrongType) storage::Partitionable::get_partition_table();
 %catches(storage::WrongNumberOfChildren, storage::DeviceHasWrongType) storage::Partitionable::get_partition_table() const;
+%catches(storage::InvalidBlockSize) storage::Region::adjust_block_size(unsigned int block_size);
```

Comment:
Honest question (I don't know how this work): is it enough for the exception to be propagated?

I mean, in the Ruby code we will likely never call `Region::adjust_block_size` directly. We will actually see the exception as a consequence of `Md::add_device`. I hope SWIG takes care of those details, but better to ask :-)

Reply:
Honestly, I donät know all the SWIG details. But it worked when I tested it from the srorage-ng side.